src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      { tweetsArray.map((tuit) =>{
```

Comment:
Podemos hacer uso de bloques de codigo pero una de las ventajas de usar arrow function es el retorno implicito como aca estas retornando una sola cosa podemos refactorizar asi 👇 
```suggestion
<div className="App">
      {tweetsArray.map((tuit, i) => (
        <Tweet key={i} tweet={tuit} />
      ))}
    </div>```